Scripts/Proxies/TwoHandedProxyBase.cs

```diff
@@ -38,35 +43,36 @@ abstract class TwoHandedProxyBase : MonoBehaviour, IProxy, IFeedbackReceiver, IS
 
         protected Transform m_LeftHand;
         protected Transform m_RightHand;
-        readonly List<ProxyFeedbackRequest> m_FeedbackRequests = new List<ProxyFeedbackRequest>();
+        readonly Dictionary<ProxyFeedbackRequest, Coroutine> m_FeedbackRequests = new Dictionary<ProxyFeedbackRequest, Coroutine>();
 
         protected Dictionary<Node, Transform> m_RayOrigins;
 
         bool m_Hidden;
+        ProxyHelper m_LeftProxyHelper;
+        ProxyHelper m_RightProxyHelper;
+        List<Transform> m_ProxyMeshRoots = new List<Transform>();
 
-        readonly Dictionary<Node, ButtonDictionary> m_Buttons = new Dictionary<Node, ButtonDictionary>();
+        ProxyFeedbackRequest m_SemitransparentLockRequest;
+        float m_ShakeFrequency;
+        Vector3 m_PreviousLeftHandPosition;
+        Vector3 m_PreviousRightHandPosition;
 
-        public Transform leftHand
-        {
-            get { return m_LeftHand; }
-        }
+        readonly Dictionary<Node, Dictionary<VRInputDevice.VRControl, List<Affordance>>> m_Affordances =
```

Comment:
Anyway we can shrink this down to an affordanceDictionary or some such?

Reply:
When you say shrink it down, do you mean create a class that derives from Dictionary, etc?

Reply:
Thoughts @mtschoen ?

Reply:
Yeah, this is what I did for ButtonDictionary. If it's just local, you can also use a class alias which doesn't affect the assembly, but either way, some kind of alias is best when the type is this long.

Also, FYI, my Unity account is @mtschoen-unity